integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/ClientPoliciesPoliciesResource.java

```diff
@@ -15,7 +16,7 @@ public interface ClientPoliciesPoliciesResource {
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    ClientPoliciesRepresentation getPolicies();
+    ClientPoliciesRepresentation getPolicies(@QueryParam("include-global-policies") Boolean includeGlobalPolicies);
```

Comment:
Is it possible to keep also original method `getPolicies()` and make it deprecated? As AFAIK we try to have backwards compatibility for admin client.

Reply:
Done! I have just added both methods.